packages/react/tests/e2e-storybook/cypress/integration/TableOfContents/TableOfContents.e2e.js

```diff
@@ -11,51 +11,357 @@
-      widths: [1280],
-    });
+describe('TableOfContents | with heading content (desktop)', () => {
+  beforeEach(() => {
+    cy.viewport(1280, 720);
+    cy.visit(`/${_paths.heading_content}`);
   });
 
-  it('should load the g10 theme', () => {
-    cy.visit(`${_path}&theme=g10`);
-    cy.viewport(1280, 780);
+  it(
+    'should load table of contents horizontal bar with links',
+    _tests.desktop.checkRender
+  );
+  it(
+    'should navigate content to selected section',
+    _tests.desktop.checkLinkFunctionality
+  );
+  it('should update current section on scroll', _tests.desktop.checkScrollSpy);
+  it(
+    'should remain visible on page throughout scroll',
+    _tests.desktop.checkStickyNav
+  );
+  it('should render correctly in all themes', _tests.all.screenshotThemes);
+});
 
-    cy.get('[data-autoid="dds--tableofcontents"]');
-    cy.wait(500);
+describe('TableOfContents | manually defined (mobile)', () => {
+  beforeEach(() => {
+    cy.viewport(320, 720);
+    cy.visit(`/${_paths.manual_default}`);
+  });
 
-    cy.screenshot();
-    // Take a snapshot for visual diffing
-    cy.percySnapshot('TableOfContents | manually defined | g10 theme', {
-      widths: [1280],
-    });
+  it(
+    'should load table of contents sidebar with links',
+    _tests.mobile.checkRender
+  );
+  it(
+    'should navigate content to selected section',
+    _tests.mobile.checkLinkFunctionality
+  );
+  it('should update current section on scroll', _tests.mobile.checkScrollSpy);
+  it(
+    'should remain visible on page throughout scroll',
+    _tests.mobile.checkStickyNav
+  );
+  it('should render correctly in all themes', _tests.all.screenshotThemes);
+});
+
+describe('TableOfContents | dynamically defined (mobile)', () => {
+  beforeEach(() => {
+    cy.viewport(320, 720);
+    cy.visit(`/${_paths.dynamic_default}`);
   });
+
+  it(
+    'should load table of contents sidebar with links',
+    _tests.mobile.checkRender
+  );
+  it(
+    'should navigate content to selected section',
+    _tests.mobile.checkLinkFunctionality
+  );
+  it('should update current section on scroll', _tests.mobile.checkScrollSpy);
+  it(
+    'should remain visible on page throughout scroll',
+    _tests.mobile.checkStickyNav
+  );
+  it('should render correctly in all themes', _tests.all.screenshotThemes);
+});
+
+describe('TableOfContents | with heading content (mobile)', () => {
+  beforeEach(() => {
+    cy.viewport(320, 720);
+    cy.visit(`/${_paths.heading_content}`);
+  });
+
+  it(
+    'should load table of contents sidebar with links',
+    _tests.mobile.checkRender
+  );
+  it(
+    'should navigate content to selected section',
+    _tests.mobile.checkLinkFunctionality
+  );
+  it('should update current section on scroll', _tests.mobile.checkScrollSpy);
+  it(
+    'should remain visible on page throughout scroll',
+    _tests.mobile.checkStickyNav
+  );
+  it('should render correctly in all themes', _tests.all.screenshotThemes);
```

Comment:
@andy-blum I think this is the issue, didn't you rename the method to `carbonThemesScreenshot`?

Reply:
talked to @andy-blum on slack and the method is calling carbonThemesScreenshot above, but i think it might be the `width` causing the error https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/7673/files#diff-a42d6ed5ec53b43667f81c546bfc457893f5157db7a2bddd00eef414126fc929R35

Reply:
fix pushed up. 🤞

Reply:
Looks like the react e2e-storybook tests are failing now 😞

Reply:
re-ran and it passed!!! Looks like this one is good to go :D